### PR TITLE
[13.0][FIX] base_vat_optional_vies: fix tests + [IMP] *: apply copier template and update pre-commit config

### PR DESCRIPTION
**Pull request by joao-p-marques**
Force EU country and compay type in test partner

After OCA/OCB@4518968#diff-1ef5fe83935572c9b9b9e6eec69a44551023f3b92fe38bed4f765a24e16694b4 and OCA/OCB@198a4dd the associated partner needs to be in a EU country and be of type `company` for the vies check to work.

@Tecnativa

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
-![Licence](https://img.shields.io/badge/licence-AGPL--3-blue.svg)
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/92/13.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-account-financial-tools-92)
-[![Build Status](https://travis-ci.org/OCA/account-financial-tools.svg?branch=13.0)](https://travis-ci.org/OCA/account-financial-tools)
-[![Coverage Status](https://coveralls.io/repos/OCA/account-financial-tools/badge.svg?branch=13.0)](https://coveralls.io/r/OCA/account-financial-tools?branch=13.0)
+[![Build Status](https://travis-ci.com/OCA/account-financial-tools.svg?branch=13.0)](https://travis-ci.com/OCA/account-financial-tools)
```
Comment (pedrobaeza) [inline README.md:2]: If this is using GH actions, Travis shouldn't be here

Comment (joao-p-marques) [reply README.md:2]: Ok, that's a bug in the template. I'll open a PR, but in the meantime we can merge this to unblock CI's...

Comment (pedrobaeza) [reply README.md:2]: OK

**Comment by pedrobaeza** [comment]
Merging manually for not triggering another build + version bump in a lot of modules

**Pull request merged**